src/msg/kava/index.js

```diff
@@ -191,70 +191,64 @@ function newMsgClaimUSDXMintingRewardVVesting(sender, receiver, multiplierName)
   }
 }
 
-function newMsgClaimHardReward(sender, multiplierName, denomsToClaim) {
+function newMsgClaimHardReward(sender, denomsToClaim) {
```

Comment:
remove multiplierName from each msg builder function

Reply:
multiplier name is still used, it's just included in the denomsToClaim now.  Users are able to select a claim factor which are set in the incentive/parameters endpoint. The user then gets to choose between a large and small (currently) rewards factor and we multiply their wards value times this factor.